shell/artifacts/Restaurants/source/RestaurantDetail.js

```diff
@@ -6,11 +6,11 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
-defineParticle(({DomParticle}) => {
+defineParticle(({DomParticle, html}) => {
 
   let host = `restaurant-detail`;
 
-  let styles = `
+  let styles = html`
```

Comment:
This file -- intentional for this PR?

Reply:
There was style leakage happening from MasterDetail to RestaurantDetail. When I fixed the former, I had to fix the latter. Neither should be in this exact PR, but they are still good changes.